examples/html-kitchen-sink/stories/index.stories.js

```diff
@@ -0,0 +1,26 @@
+import { document } from 'global';
+import { storiesOf } from '@storybook/html';
+import { action } from '@storybook/addon-actions';
+import { linkTo } from '@storybook/addon-links';
+
+import './welcome.css';
+import welcome from './welcome.html';
+
+document.addEventListener('click', e => {
+  const { kind, story } = e.target.dataset;
+  if (kind || story) {
+    e.preventDefault();
+    linkTo(kind, story)();
+  }
+});
```

Comment:
We could incorporate something like this into links addon

Reply:
I can't see how it will work with a real-world example, because probably your components won't be coupled to the "story"/"kind"

Reply:
We can use some namespace, like `data-sb-story`

Reply:
The idea is, if you want a link to other story, you just add corresponding data-attribute to the element that you want to be the link